lib/organisms/gravity-form/_style.scss

```diff
@@ -0,0 +1,50 @@
+.ln-o-gravity-form {
+
+  .hidden {
+    display: none !important;
+  }
+
+  &-header {
```

Comment:
It's not required, but - I just thought you wanted some default styling in there? Or will that be taken care of elsewhere?

I can just take all that out if you don't want it :)